Documentation/how-to-build-and-run-ilcompiler-in-console-shell-prompt.md

```diff
@@ -20,6 +20,7 @@ To consume the CLI tools installed as part of the build, do the following:
 
 * Add `<repo_root>\bin\tools\cli\bin` to the path
 * set `DOTNET_HOME` environment variable to `<repo_root>\bin\tools\cli`
+* On windows ensure you are using a VS developer X64 command prompt
```

Comment:
Ah I had no idea there were two different command prompts. I have updated to 'VS2015 x64 Native Tools Command Prompt' which is the exact text shown when I search for the command in the windows search box, as well as the title of the window after launching.